core/common/src/main/java/alluxio/Configuration.java

```diff
@@ -95,6 +96,25 @@ public static void defaultInit() {
         String.valueOf(ChannelType.defaultType()));
     defaultProps.setProperty(PropertyKey.USER_NETWORK_NETTY_CHANNEL.toString(),
         String.valueOf(ChannelType.defaultType()));
+    // Set ramdisk volume according to OS type
```

Comment:
this method is getting pretty long, do you think it makes sense to have a helper method for creating default properties?